src/Controller/BlogController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class BlogController extends AbstractController
+{
+    /**
+     * @Route("/blog/{slug}", name="blog_show")
+     */
+    public function show(BlogPost $post)
+    {
+        // $slug will equal the dynamic part of the URL
+        // e.g. at /blog/yay-routing, then $slug='yay-routing'
+
+        // ...
+    }
+
+    /**
+     * @Route("/blog/{page<\d+>?1}", name="blog_list", priority="2")
+     */
+    public function list(int $page)
+    {
+        // ...
+    }
+}
```

Comment:
EOL

Reply:
I've updated the commit and fixed this issue, please check out the new commit content.